waitress/urls.py

```diff
@@ -18,6 +18,7 @@
 urlpatterns = [
     path("", LoginHandler.as_view(), name="login"),
     path("dashboard", Dashboard.as_view(), name="dashboard"),
+    path("api/v1/users", UserHandler.as_view(), name="users"),
```

Comment:
Not sure what this is suppose to do yet

Reply:
This PR isn't done yet, the purpose of me creating this and tagging you all is to have this discussion.

Going forward I think I'm leaning towards having a separate SPA for the dashboard so as to reduce load on the server. Also it appears there's a plan to make use of the waitress app in other countries, not sure what the process is but I'm meeting with Kes tomorrow.

Let me know if you want to be involved @abiodun0

Reply:
If this is supposed to be used in other locations, then my advice is to decouple it as much as possible

Have a standalone service that just provides the API, have a separate SPAs for the different dashboard

Even the api makes sense because that's where the mobile clients will be making requests too